backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java

```diff
@@ -189,19 +167,14 @@ private void deleteExperiments(Session session) {
       try {
         var transaction = session.beginTransaction();
         var updateDeletedStatusExperimentRunQueryString =
-            new StringBuilder("UPDATE ")
-                .append(ExperimentRunEntity.class.getSimpleName())
-                .append(" expr ")
-                .append("SET expr.")
-                .append(ModelDBConstants.DELETED)
-                .append(" = :deleted ")
-                .append(" WHERE expr.")
+            new StringBuilder(
```

Comment:
Same here; finish converting fully to a string as opposed to a combo of StringBuilder and String

Reply:
Updated to string instead of string builder